src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -2114,7 +2463,10 @@ JNI_METHOD(void, invoke)
     tlvBytesObj = static_cast<jbyteArray>(env->CallObjectMethod(invokeElement, getTlvByteArrayMethod));
     VerifyOrExit(!env->ExceptionCheck(), err = CHIP_JNI_ERROR_EXCEPTION_THROWN);
     {
-        app::CommandPathParams path(static_cast<EndpointId>(endpointId), /* group id */ 0, static_cast<ClusterId>(clusterId),
+        GroupId groupId = device->GetSecureSession().Value()->IsGroupSession()
+            ? device->GetSecureSession().Value()->AsOutgoingGroupSession()->GetGroupId()
+            : chip::kUndefinedGroupId;
+        app::CommandPathParams path(static_cast<EndpointId>(endpointId), groupId, static_cast<ClusterId>(clusterId),
```

Comment:
see the below constructor, if this is group communication, you should not set endpointId, you could use uint16_t variable, then set it for endpointId or group based upon whether it is group communication
 CommandPathParams(uint16_t aId, ClusterId aClusterId, CommandId aCommandId, const BitFlags<CommandPathFlags> & aFlags) :
        mClusterId(aClusterId), mCommandId(aCommandId), mFlags(aFlags)
    {
        if (aFlags == CommandPathFlags::kEndpointIdValid)
        {
            mEndpointId = aId;
        }
        else if (aFlags == CommandPathFlags::kGroupIdValid)
        {
            mGroupId = aId;
        }
    }

Reply:
I understand in your comment. I updated some codes. (Java, JNI Layer).